operator/controllers/kyma_controller.go

```diff
@@ -191,10 +193,13 @@ func (r *KymaReconciler) ReconcileFromConfigMap(ctx context.Context, req ctrl.Re
 						"operator.kyma-project.io/release":         progression.New,
 					},
 				},
-				"spec": componentYaml["spec"],
+				"spec":   componentYaml["spec"],
+				"status": map[string]string{},
```

Comment:
this breaks on deep copy, you will have to use `interface{}` here
```suggestion
				"status": map[string]interface{},
```

Reply:
I removed it completely, since we do not need an empty map anyways